app/code/community/Demac/MultiLocationInventory/Model/CatalogInventory/Resource/Stock/Item.php

```diff
@@ -16,12 +16,44 @@ class Demac_MultiLocationInventory_Model_CatalogInventory_Resource_Stock_Item ex
      */
     private function _loadByProductAndStoreId(Mage_CatalogInventory_Model_Stock_Item $item, $productId, $storeId)
     {
+        $arrDefaultData = array(
+            'product_id'                  => $productId,
+            'stock_id'                    => 0,
+            'qty'                         => 0.0000,
+            'min_qty'                     => 0.0000,
+            'use_config_min_qty'          => 1,
+            'is_qty_decimal'              => 0,
+            'backorders'                  => 0,
+            'use_config_backorders'       => 1,
+            'min_sale_qty'                => 1.0000,
+            'use_config_min_sale_qty'     => 1,
+            'max_sale_qty'                => 0.0000,
+            'use_config_max_sale_qty'     => 1,
+            'is_in_stock'                 => 0,
+            'low_stock_date'              => null,
+            'notify_stock_qty'            => null,
+            'use_config_notify_stock_qty' => 1,
+            'manage_stock'                => 0,
+            'use_config_manage_stock'     => 1,
+            'stock_status_changed_auto'   => 0,
+            'use_config_qty_increments'   => 1,
+            'qty_increments'              => 0.0000,
+            'use_config_enable_qty_inc'   => 1,
+            'enable_qty_increments'       => 0,
+            'is_decimal_divided'          => 0
+        );
+
+        // Only configurable products seem to be in the
```

Comment:
Added a default override when for when the result below is empty yet there is data in the demac stock table.